textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -1,6 +1,10 @@
 package edu.uci.ics.textdb.dataflow.queryrewriter;
 
+import edu.uci.ics.textdb.api.common.IField;
```

Comment:
I saw your implementation, containsAllResults() also works when there are two lists of Strings to be compared. 

Instead of forming Tuples, you could use that method to compare your "rewrittenStrings" and "expectedRewrittenStrings".